cypress/e2e/nodes/Preview.spec.js

```diff
@@ -129,6 +129,21 @@ describe('Preview extension', { retries: 0 }, () => {
 
 	})
 
+	describe('insertPreview command', { retries: 0 }, () => {
+
+		it('is available in commands', () => {
+			expect(editor.commands).to.have.property('insertPreview')
+		})
+
+		it('inserts a preview', () => {
+			editor.commands.clearContent()
+			editor.commands.insertPreview('https://nextcloud.com')
+			editor.commands.setTextSelection(1)
```

Comment:
Can we have a comment on why this is needed? Just from reading I don't get why ;)

Reply:
The `expectPreview()` command expects the selection to be set. For all other tests, this is done in `prepareEditor()`, but we cannot use that because we want an empty editor to start with.